src/status_im/chat/models/message.cljs

```diff
@@ -186,7 +186,7 @@
 
           :else
           (merge {:send-message (assoc-in options [:message :to] chat-id)}
-                 (when-not command) {:send-notification fcm-token}))))))
+                 (when fcm-token {:send-notification fcm-token})))))))
```

Comment:
isn't it going to send a notification for commands ? which was not the case before ?

Reply:
@yenda is correct, checking the previous code when it was working (release `0.9.13`) the notification was sent only for plain 1-1 messages, so it should be (in the `:else` branch):
```clj
(cond-> {:send-message (assoc-in options [:message :to] chat-id)}
  fcm-token (assoc :send-notification fcm-token))
```